Automattic/pocket-casts-android
# End of Year: Number of listened podcasts/episodes story design

| 📘 Project: #410 |
|:---:|

iOS PR: https://github.com/Automattic/pocket-casts-ios/pull/454

## Description
Add the design for the "number of podcasts/episodes listened" story.

The podcasts cover displayed is determined by getting the top 10 podcasts and reversing it. We then display the last 6 available.

## Testing Instructions
1. Set `END_OF_YEAR_ENABLED` feature flag to `true` in `base.gradle`
2. Run the app and make sure you're logged in to an account with a few items on Listening History
3. When the prompt appears, tap "View My 2022"
4. Navigate to the 5th story
4. ✅ Check the design

## Screenshots or Screencast 

Phone (Pixel 6)  | Tablet (Nexus 7 - 2012)
-----| ------
<img width=320 src="https://user-images.githubusercontent.com/1405144/201460468-ec72c68a-5a9a-4ddd-9a84-9adeb5d20577.png"/> | <img width=500 src="https://user-images.githubusercontent.com/1405144/201460501-2540dbdf-6fff-4acf-9c32-79c3418483c0.png"/>






## Checklist
- If this is a user-facing change, I have added an entry in CHANGELOG.md
- I have considered whether it makes sense to add tests for my changes
- Any jetpack compose components I added or changed are covered by compose previews
 
#### I have tested any UI changes...
